CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Master
 
+* BUGFIX: xos model deos not modify config on legacy Extreme Networks devices (sq9mev)
```

Comment:
This looks like a typo (does?), also the sentence may be clearer if worded as 'no longer unintentionally modifies' or similar.